src/App/App.js

```diff
@@ -66,9 +83,16 @@ class App extends Component {
       }
     })
   }
-
-  logIn() {
-    this.setState({pageDisplayed: 'home', isLoggedIn: true})
+  
+  logIn = async (user) => {
+    const ratings = await getAllUserRatings(user.user.id)
+    this.setState({
+      pageDisplayed: 'home', 
+      isLoggedIn: true, 
+      userId: user.user.id, 
+      userName: user.user.name,
+      userRatings: ratings
+    })
```

Comment:
You can cut down the size of your state by just storing the entire user object. That way, you can see if the user is logged in by checking to see if the user exists in state (over isLoggedIn). Then you can reference user.id and user.name for the userId and userName props.